java/runtime/src/main/java/org/ray/runtime/config/RayConfig.java

```diff
@@ -104,11 +104,6 @@ public RayConfig(Config config) {
             + "setting it to the number of CPU cores: {}", numCpu);
         resources.put("CPU", numCpu * 1.0);
       }
-      // TODO(romilb): This needs to be fixed to work with new resource semantics
```

Comment:
Hi @raulchen - Java tests are failing with this new PR. I removed this line to avoid creating any GPU resources with capacity zero, but still while initializing ray, some code is setting CPU capacity to 0. Do you have any pointers on where this could be happening?

Test logs: https://travis-ci.com/ray-project/ray/jobs/192077053#L590